integrations/symfony-bundle.rst

```diff
@@ -342,6 +343,30 @@ The only steps they need is ``require`` one of the adapter implementations in
 their projects ``composer.json`` and instantiating the ``HttplugBundle`` in
 their kernel.
 
+Mock responses in functional tests
+``````````````````````````````````
```

Comment:
can you please addd a paragraph here that mentions again that you need the separate mock client library, and cross-link that to the clients/mock-client doc?